doc/RFC-MMP.llnl.04.txt

```diff
@@ -0,0 +1,913 @@
+=====================================================================
+2016-07-28 Draft 4		ZFS Multi-{Modifier,Mount} Protection
+=====================================================================
+Originally created 2009 by ricardo.correia@oracle.com
+  ** references 1
+Revised 2017 by {faaland1,bass6,behlendorf1}@llnl.gov
+=====================================================================
+
+As stated in RFE 6710718 (multi-import protection for the DMU), it would be
```

Comment:
OK.  Yes, I would like to put this document somewhere accessible, but don't see a directory for such artifacts in the zfsonlinux repo.  Do you have a place for such documents in Illumos?